### DISCUSSION TITLE
Agile Export Rates Unknown

### DISCUSSION THREAD

**Discussion by JRascagneres** [BottlecapDave/HomeAssistant-OctopusEnergy#600] [Q&A]
Hi, I just got onto Octopus Export and its popped up as a device. However the event data is 'unknown' any ideas?:
![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/3264971/f380fc99-665e-4a72-8366-c2ddd496d5eb)

**Comment by BottlecapDave** [comment]
Sorry for the late response. Did it just take a little while to populate?

**Reply by JRascagneres** [reply]
Yup, thats why I closed it. Seemed to take a little while but eventually sorted itself. My assumption is it was waiting until the next hour to trigger or something and didn't do it on demand... This is purely a guess though, I didn't look much at it.
